using-mobile-extensions/adobe-campaign-standard/adobe-campaignclassic-api-reference.md

```diff
@@ -0,0 +1,70 @@
+# Campaign API reference
+
+## Set linkage fields
+
+Allows Campaign to connect fields from separate databases and create a more developed and personalized messaging experience. Profile template-based messages that contain PII-based personalization are downloaded. 
+
+{% tabs %}
+
+{% tab title="iOS" %}
+### setLinkageFields
+
+
+#### Syntax
+
+```objectivec
++ (void) setLinkageFields: (nonnull NSDictionary<NSString*, NSString*>*) linkageFields;
+```
+
+#### Objective-C
+
+The **linkage fields** are stored as a base64-encoded JSON string in memory and sent in a custom HTTP header 'X-InApp-Auth' in all future Campaign rules download requests until ACPCampaign::resetLinkageFields is invoked.
```

Comment:
We also need to specify that linkage fields are cleared if the app is no longer running (graceful close or crash). The linkage fields are also cleared if privacy status is changed to opt out.

Reply:
Also why do we have this description in objective-c section. This is a common behavior and can be moved to top (line 5)